src/VDPRegViewer.cpp

```diff
@@ -35,20 +35,27 @@ void buttonHighlightDispatcher::receiveState(bool state)
 
 VDPRegViewer::VDPRegViewer(QWidget *parent)
 	: QDialog(parent)
+	, regs{0}
```

Comment:
This is to zero-initialize the full array?
Maybe write it like this in the .h file:
```uint8_t regs[64 + 16 + 2 + 3] = {};```
This keeps declaration and initialization close together (on the same line, instead of spread over two files).